darwin/exporter/formats/__init__.py

```diff
@@ -13,4 +13,5 @@
     "semantic_mask_grey",
     "semantic_mask_index",
     "yolo",
+    "nifti",
```

Comment:
Did you also update the docs as the comment on darwin/exporter/formats/__init__.py:5 says?

If not, can you, or can you confirm it's either not needed or open a ticket.

Reply:
Yes, the docs were updated!